imap_processing/ultra/l1b/ultra_l1b_extended.py

```diff
@@ -666,106 +668,62 @@ def get_ctof(tof: np.ndarray, path_length: np.ndarray, type: str) -> NDArray:
     path_length : np.ndarray
         Path length (r) (hundredths of a millimeter).
     type : str
-        Type of event, either "ph" or "ssd".
+        Type of event, either "PH" or "SSD".
 
     Returns
     -------
     ctof : np.ndarray
         Corrected TOF (tenths of a ns).
+    magnitude_v : np.ndarray
+        Magnitude of the particle velocity (tenths of a ns / mm).
     """
     dmin_ctof = getattr(UltraConstants, f"DMIN_{type}_CTOF")
 
     # Multiply times 100 to convert to hundredths of a millimeter.
     ctof = tof * dmin_ctof * 100 / path_length
 
-    return ctof
+    # Convert from mm/0.1ns to km/s.
+    magnitude_v = dmin_ctof / ctof * 1e4
```

Comment:
Double check the conversion... 
mm -> km = 1e-3
1/0.1ns -> 1/s = 1e10 (?)
mm/0.1ns -> km/s = 1e-3 * 1e10 = 1e7. 
Maybe I'm doing it wrong but I thought that 1e4 seemed off.

Reply:
@subagonsouth - you're missing a factor of 1000 converting mm ($10^{-3}$) to km ($10^{3}$).

$\frac{1}{0.1 } \frac{mm}{ns} = \frac{10^{-3}}{0.1 * 10^{-9}} \frac{m}{s} = \frac{10^{-3}}{ 10^{-10}}  \frac{m}{s} = 10^{7}  \frac{m}{s} = 10^{4}  \frac{km}{s}$ 

P.S. Clearly I have way too much fun with LaTEX 😁